RealmModelGenerator/EntitiesView.swift

```diff
@@ -8,24 +8,39 @@
 
 import Cocoa
 
+protocol EntitiesViewDataSource : NSObjectProtocol {
+    func numberOfRowsInEntitiesView(entitiesView:EntitiesView) -> Int
+    func entitiesView(tableView:NSTableView, viewForTableColmn tableColumn: NSTableColumn?, row: Int) -> NSView?
```

Comment:
rename to entitiesView(entitiesView:EntitiesView tittleForEntityAtIndex index:Int) -> String?